kuma/users/jinja2/users/user_edit.html

```diff
@@ -12,135 +12,141 @@
     </div>
+    {% endif %}
+
+    <form class="submission" method="post" action="">
+      {% csrf_token %}
+
+      <fieldset class="section notitle" id="personal">
+        <ul>
+          <li id="field-beta" class="field">
+            <label for="{{ user_form.beta.id_for_label }}">{{ user_form.beta.label }}</label> {{ user_form.beta }}
+            <p class="field-note">{{ _("We'd love to have your feedback on site changes! Beta testers get access to new features first and we send the occasional email asking for help testing specific things.") }}</p>
+          </li>
+          <li id="field_email" class="field type_email required">
+            <label>{{ _('Primary Email') }} <a class="field-management" href="{{ url('account_email') }}">{{ _('Edit email') }}</a></label>
+            {{ edit_user.email }}
+          </li>
+          {{ li_field(user_form, 'username') }}
+          {{ li_field(user_form, 'fullname') }}
+          {{ li_field(user_form, 'title') }}
+          {{ li_field(user_form, 'organization') }}
+          {{ li_field(user_form, 'location') }}
+          {{ li_field(user_form, 'locale') }}
+          {{ li_field(user_form, 'timezone') }}
+          {{ li_field(user_form, 'irc_nickname') }}
+
+          <li class="{{ ' error' if user_form.interests.errors }}">
+            <label for="{{ user_form.interests.id_for_label }}">{{ _('Interests (tags)') }}</label>
+            <p class="field-note">{{ _('Separate tags with commas or spaces. Join multi-word tags with double quotes, like "web standards".') }}</p>
+            {{ user_form.interests }}
+            {{ user_form.interests.errors }}
+          </li>
+
+          <li class="{{ ' error' if user_form.expertise.errors }}">
+            <label for="{{ user_form.expertise.id_for_label }}">{{ _('Areas of expertise') }}</label>
+            <p class="field-note">{{ _('Add your interests first, then declare yourself an expert in selected topics.') }}</p>
+            {{ user_form.expertise }}
+            {{ user_form.expertise.errors }}
+          </li>
+        </ul>
+      </fieldset>
+
+      <fieldset class="section" id="users">
+        <legend><b>{{ _('My Profiles') }}</b></legend>
+        <ul>
+
+          <li id="field_github_url" class="field site github">
+            {% if accounts.github %}
+            <label for="{{ user_form.github_url.id_for_label }}"><i class="icon-github" aria-hidden="true"></i>GitHub</label>
+            <div class="field-account">
+              {% for github_account in accounts.github %}{% if github_account.extra_data %}{{ github_account.get_profile_url() }}{% if not loop.last %}, {% endif %}{% endif %}{% endfor %}
+            </div>
+
+            <div id="field_is_github_url_public">
+              <label for="{{ user_form.is_github_url_public.id_for_label }}">
+                {{ user_form.is_github_url_public.label }}
+              </label> {{ user_form.is_github_url_public }}
+            </div>
```

Comment:
It's hard to tell because of the indenting change, but the non-indenting change is adding this new field, which looks 👍 . This is why I try to change indenting in once commit, and change code in the next.